File: 65472.java
public class Main {
    public static void main(String[] args) {
        IntStream.of(6, 5, 4, 7, 2)
                .mapToObj(i -> (i % 2 != 0 ? "🟦" : "🟥")
                        .repeat(i))
                .forEach(System.out::println);
    }
}

File: 65472.rs
fn main() {
    for (i, j) in [6, 5, 4, 7, 2].iter().enumerate() {
        (0..*j).for_each(|_| print!("{}", if i % 2 == 0 { "🟥" } else { "🟦" }));
        println!();
    }
}
